packaging/kibana.spec

```diff
@@ -22,16 +22,32 @@ cd %_builddir
 rm -rf %{name}
 mkdir %{name}
 cd %{name}
+
 #extract sources
 tar xzf %_sourcedir/%{name}-%{version}.tar.gz
 if [ $? -ne 0 ]; then
    exit $?
 fi
 
+#extract dlumbrer/kbn_network plugin, following the renaming and file modification steps from github repo
+unzip -q resources/plugins/kbn_network*.zip -d plugins/
+if [ $? -ne 0 ]; then
+   exit $?
+fi
+mv plugins/kbn_network* plugins/network_vis
```

Comment:
Do you need to do a `mkdir` of `plugins/network_vis` first?

Reply:
No, this is using `mv` to rename a directory to `network_vis`, not to move it under `network_vis`. It does rely on not having a directory named `network_vis` before the `mv` though - otherwise it would nest the directories. (But that's fine as long as we're always extracting the plugin and not storing it in kibana/plugins/network_vis)